tests/validation/tests/v3_api/test_cis_scan.py

```diff
@@ -367,3 +405,31 @@ def wait_for_cis_pod_remove(cluster,
                                 name="security-scan-runner-" +
                                      cluster_scan_report_id)
         time.sleep(.5)
+
+
+def create_cluster_cis(scan_tool_version="rke-cis-1.4"):
+    aws_nodes = \
+        AmazonWebServices().create_multiple_nodes(
+            5, random_test_name(HOST_NAME))
```

Comment:
I'm not a fan of this part hardcode creating the 5 nodes because some of the tests in this module just need 3 but this will be handled [by this other PR](https://github.com/rancher/rancher/pull/27452/files#diff-21c77191f3e5078588f03cebfd82925aR295). Not a biggie.